mayastor/src/grpc/server.rs

```diff
@@ -19,18 +25,24 @@ impl MayastorGrpcServer {
         rpc_addr: String,
     ) -> Result<(), ()> {
         info!("gRPC server configured at address {}", endpoint);
+        let address = Cow::from(rpc_addr);
         let svc = Server::builder()
             .add_service(MayastorRpcServer::new(MayastorSvc::new(
                 Duration::from_millis(4),
             )))
             .add_service(BdevRpcServer::new(BdevSvc::new()))
-            .add_service(JsonRpcServer::new(JsonRpcSvc {
-                rpc_addr,
-            }))
+            .add_service(v1::bdev::BdevRpcServer::new(BdevService::new()))
+            .add_service(JsonRpcServer::new(JsonRpcSvc::new(address.clone())))
+            .add_service(v1::json::JsonRpcServer::new(JsonService::new(
+                address.clone(),
+            )))
             .serve(endpoint);
 
         match svc.await {
-            Ok(_) => Ok(()),
+            Ok(result) => {
+                trace!(?result);
```

Comment:
Does this create "duplicate" traces? I think some of the rpc already trace their results somehow.

Reply:
Didnt get this comment @tiagolobocastro

Reply:
what I meant was that some of the service handlers already output some info, some maybe this wasn't needed, eg:
`info!("Added child to nexus {}", uuid);`
But actually it might be worth having the whole reply result for TraceLevel, so yeah maybe keep it then :+1: